crates/bevy_time/src/timer.rs

```diff
@@ -329,7 +336,11 @@ impl Timer {
     /// ```
     #[inline]
     pub fn percent(&self) -> f32 {
-        self.elapsed().as_secs_f32() / self.duration().as_secs_f32()
+        if self.duration == Duration::ZERO {
+            1.0
+        } else {
+            self.elapsed().as_secs_f32() / self.duration().as_secs_f32()
+        }
```

Comment:
style question: why do the check yourself here and not use `checked_div` like above?

Reply:
f32 can be divided by `0.0` without panicking. It returns `NaN`.
 
I tried calling `checked_div`, but compiler argues about non existing method:

```
error[E0599]: no method named `checked_div` found for type `f32` in the current scope
```

Reply:
oh right, you would have to use `as_secs` to get u64 and then another call to get the milliseconds, would be a pain...